File: ss-won/programmers/17681.py
def solution(n, arr1, arr2):
    answer = []
    for i in range(n):
        li = arr1[i] | arr2[i]
        li = str(bin(li))[2:]
        res = ""
        if len(li) < n:
            li = "".join(['0' for _ in range(n-len(li))]) + li
        for idx in range(n):
            if li[idx] == '1':
                res += '#'
            else:
                res += ' '
        answer.append(res)
    return answer

File: ss-won/programmers/17681.js
const solution = (n, arr1, arr2) => {
  let answer = [];
  let barr1 = arr1
    .map((v) => v.toString(2).split(""))
    .map((v) => {
      while (v.length !== n) {
        v.unshift("0");
      }
      v = v.map((val) => val * 1);
      return v;
    });
  let barr2 = arr2
    .map((v) => v.toString(2).split(""))
    .map((v) => {
      while (v.length !== n) {
        v.unshift("0");
      }
      v = v.map((val) => val * 1);
      return v;
    });

  answer = barr1.map((val, idx) => {
    return val.map((v, i) => (v > barr2[idx][i] ? v : barr2[idx][i]));
  });

  answer = answer.map((val) => val.map((v) => (v ? "#" : " ")).join(""));
  return answer;
};
